examples/asr/conf/speech_to_text_hf_finetune.yaml

```diff
@@ -0,0 +1,175 @@
+name: "Speech_To_Text_HF_Finetuning"
```

Comment:
This shouldn't be at the root of the conf dir, create a finetune dir, of a HF dir

Reply:
done